src/PageLayout/PageLayout.tsx

```diff
@@ -423,6 +488,8 @@ const Pane = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageLayout
       positionWhenNarrow = 'inherit',
       width = 'medium',
       padding = 'none',
+      resizable = false,
+      paneWidthStorageKey = 'paneWidth',
```

Comment:
Since this is a prop on the `Pane` component, including `pane` in the prop name feels redundant. How do you feel about calling this `widthStorageKey`?
```suggestion
      widthStorageKey = 'paneWidth',
```

Reply:
Yes, good point. Done in https://github.com/primer/react/pull/2352/commits/7cfccdbb231ccfb3c3c0d2e69e2c16b4dd525d4d